cli/command/engine/init.go

```diff
@@ -1,62 +1,10 @@
 package engine
 
 import (
-	"context"
-
-	"github.com/docker/cli/cli"
-	"github.com/docker/cli/cli/command"
 	clitypes "github.com/docker/cli/types"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
 )
 
 type extendedEngineInitOptions struct {
```

Comment:
I guess this type could be renamed as engineUpdateOptions, and moved to the update command, so you can remove this file?